src/Constants/Theme/Colors.ts

```diff
@@ -522,10 +530,13 @@ const light: Colors = {
         buttonTextDisabled: COLORS.GREY_400,
     },
     actionBottomSheet: {
-        background: COLORS.GREY_50,
+        background: COLORS.WHITE,
         border: COLORS.GREY_200,
         isActiveBackground: COLORS.GREY_100,
-        iconBackground: COLORS.GREY_100,
+        iconBackground: {
+            background: COLORS.TRANSPARENT,
```

Comment:
Same applies here: if you change this color, it could break the color styles of that component above